headlineView.js

```diff
@@ -43,6 +43,7 @@ class HeadlineView {
       a.title = story['headline'];
       a.href = story['url'];
       div.append(a);
+      div.append(document.createElement("br"));
       this.mainContainerEl.append(div);
     });
   }
```

Comment:
Excellent event listeners; good method for resting the page to load the search results.